CaPPMS/CaPPMS.csproj

```diff
@@ -26,11 +26,17 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="5.0.8" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="5.0.9" NoWarn="NU1605" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="5.0.8" NoWarn="NU1605" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.9" />
```

Comment:
There is not a backing database by design

Reply:
I'm actually not sure what this file is exactly, I just know that when I opened vs, it changed it. I could just not include this file's changes.